.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      # Don't bail all runs so that we can learn if we introduced a feature that only works in
+      # a more recent Node.js version
+      fail-fast: false
+      matrix:
+        node: [12, 14, 16]
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Test
+        run: npm test
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Lint
+        run: npm run lint
+
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Build docs
+        run: npm run docs:build
```

Comment:
Do we build docs just to check that nothing fails?

Reply:
Pretty much, I think.

I don't recall why I added it there back in [`f9aec48` (#579)](https://github.com/express-validator/express-validator/pull/579/commits/f9aec483c4944dbc1c8218fb5766ba886b377542).